src/consensus/pbft/libbyz/replica.cpp

```diff
@@ -1165,6 +1166,7 @@ void Replica::send_prepare(Seqno seqno, std::optional<ByzInfo> byz_info)
 
 void Replica::send_commit(Seqno s, bool send_only_to_self)
 {
+  LOG_DEBUG_FMT("Sending commit for seqno: {}", s);
```

Comment:
Is this going to be very verbose?

Reply:
I keep adding this every time I debug pbft. It is a good indicator of what state the replica is in if it does send a commit message but then it doesn't add the commit message to it's certificate, so then you don't see the next debug message in `Replica::send_commit` (https://github.com/microsoft/CCF/blob/master/src/consensus/pbft/libbyz/replica.cpp#L1188)

Reply:
Things that are per-transaction or more often should be TRACE, this is presumably per-batch so DEBUG or TRACE should be fine.

Reply:
Please use trace